scraper/src/youtube2zim/youtube.py

```diff
@@ -335,6 +459,25 @@ def extract_playlists_details_from(collection_type, youtube_id):
 
         # retrieve list of playlists for that channel
         playlist_ids = [p["id"] for p in get_channel_playlists_json(main_channel_id)]
+        
+        # Retrieve the shorts,long videos and lives playlist ID
+        user_long_uploads_playlist_id = get_user_long_uploads_playlist_id(main_channel_id)
+        user_short_uploads_playlist_id = get_user_short_uploads_playlist_id(main_channel_id)
+        user_lives_playlist_id = get_user_lives_playlist_id(main_channel_id)
+        
+
+        if user_long_uploads_playlist_id is not None:
```

Comment:
You can replace this whole block by `playlist_ids += filter(None, [user_long_uploads_playlist_id, user_short_uploads_playlist_id, user_lives_playlist_id])` and a comment `Add special playlists if they exists`